package.json

```diff
@@ -89,11 +90,11 @@
     "@types/jest": "^29.1.2",
     "babel-plugin-i18next-extract": "^0.9.0",
     "eslint-config-prettier": "^8.5.0",
-    "husky": "^2.7.0",
+    "husky": "^8.0.1",
```

Comment:
Updating husky across these versions will break linting, which is the reason, why this is still 2.7

Reply:
Ok according to confluence the reason was:

> Because Debian oldstable and stable don’t provide a git package newer then 2.20.1 husky should only get updates for patch versions if there will be any.